src/pages/home/report/ReportActionItemReimbursed.js

```diff
@@ -0,0 +1,27 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import Text from '../../../components/Text';
+import styles from '../../../styles/styles';
+import withLocalize, {withLocalizePropTypes} from '../../../components/withLocalize';
+
+const propTypes = {
+    amount: PropTypes.string.isRequired,
+    submitterDisplayName: PropTypes.string.isRequired,
+    isFromSubmitterAddingBankAccount: PropTypes.bool.isRequired,
+
+    ...withLocalizePropTypes,
+};
+
+function ReportActionItemReimbursed(props) {
+    return (
+        <Text style={[styles.chatItemMessage, styles.colorMuted]}>
+            {props.translate('iou.payerSettledUp', {amount: props.amount})}{' '}
+            {props.isFromSubmitterAddingBankAccount ? props.translate('iou.afterAddedBankAccount', {submitterDisplayName: props.submitterDisplayName}) : ''}
+        </Text>
+    );
+}
```

Comment:
I see that this is coming from the design doc but I am curious if we should have two actions for the "Paid $40" logic. cc @luacmartins what do you think?

Reply:
https://docs.google.com/document/d/17jFcMsAotXvsMhXivoF3u7fyzeyXkKpRGXR88Ud0oWY/edit?disco=AAAAzP95sRQ